### PR TITLE
[A.Zhukevych]Interactive-task1

### PR DESCRIPTION
**Pull request by Hollister009**

### CODE REVIEW

**Comment by Hollister009** [comment]
https://hollister009.github.io/ui-internship-lwo2/css-practice/task2/

File: css-practice/task2/js/script.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+(function(doc) {
+  // 1. Responsive menu section
+  const menuBtn = doc.getElementById('menu-btn');
+  const navMenu = doc.getElementById('menu');
+  let menuShown = false; // navMenu State
+
+  const toggleMenu = (e) => {
+    e.preventDefault();
+    if (!menuShown) {
```
Comment (AlyonaPianykh) [inline css-practice/task2/js/script.js]: Classlist has method 'contains' that returns true or false depending if class is in classlist. So you don't need extra variable `menuShown `. By the way, it also has method 'toggle' that can be used in your case

Comment (Hollister009) [reply css-practice/task2/js/script.js]: totally forgot 🤦‍♂️

File: css-practice/task2/js/script.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  };
+
+  const displayMsg = (array) => {
+    return array.forEach((input) => {
+      if (input.tagName !== 'TEXTAREA') {
+        const validityTip = input.parentNode.lastElementChild;
+        if (!input.value) {
+          validityTip.textContent = isEmptyMsg;
+        } else if (input.getAttribute('isvalid') === 'false') {
+          if (input.name === 'name') validityTip.textContent = notNameMsg;
```
Comment (AlyonaPianykh) [inline css-practice/task2/js/script.js]: Create one variable called message, set value to this message in switch block

File: css-practice/task2/js/script.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+          if (input.name === 'email') validityTip.textContent = notEmailMsg;
+          if (input.name === 'password') validityTip.textContent = notPassMsg;
+        } else {
+          return;
+        }
+        validityTip.style.display = 'block';
+        const removeTip = () => {
```
Comment (AlyonaPianykh) [inline css-practice/task2/js/script.js]: Create separate function for clearing tip

File: css-practice/task2/js/script.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      return input.getAttribute('isvalid') === 'true';
+    });
+    if (!preSubmit) {
+      displayMsg(array);
+      return;
+    }
+    alert('Your data was submitted');
+  };
+
+  const displayMsg = (array) => {
```
Comment (AlyonaPianykh) [inline css-practice/task2/js/script.js]: This method looks ugly, please refactor it

File: css-practice/task2/src/scss/_helper.scss [hunk 5]
```diff
@@ -40,3 +42,30 @@
...
+
+%input-contact {
+  width: 100%;
+  border: none;
+  padding: 0 1em;
+  line-height: 2;
+  border-bottom: 1px solid rgba($color-black, .15);
+
+  &:focus {
+    background-color: #ffffcc;
```
[the remark this answers is not in the document]
Comment (Hollister009) [reply css-practice/task2/src/scss/_helper.scss]: Agree, just pure laziness 🤓

**Pull request merged**